tests/contrib/patch.py

```diff
@@ -321,16 +321,20 @@ def test_patch_import(self):
             """
             The integration should test that each class, method or function that
             is to be patched is in fact done so when ddtrace.patch() is called
-            after the module is imported.
+            after the module is imported. In addition, we check that the integration has
+            not been imported due to patching.
+
+            For example:
 
             an appropriate ``test_patch_import`` would be::
 
                 import redis
                 ddtrace.patch(redis=True)
                 self.assert_module_patched(redis)
             """
-            module = importlib.import_module(self.__module_name__)
             self.__patch_func__()
+            self.assert_not_module_imported(self.__module_name__)
```

Comment:
One important lesson from the Django integration is that whilst we were triggering the patching when `django` was imported, that patching force-imported Django submodules. So checking for just the required module might not be enough. Something better is to check for any modules prefixed with `{self.__module_name__.}`, but even this might not be enough (Django imported the `channels` module if available, for instance)

Reply:
Hmmm I can't think of a clever way to do that then, since each integration is loading stuff besides the module it's patching anyways. Is there any way to track what a module was loaded by? I looked around and couldn't find anything. Maybe we go with a `startswith()` to begin?